Description: Mark test_write_or_driver_error as flaky.
Author: Bas Couwenberg <sebastic@debian.org>
Bug: https://github.com/Toblerity/Fiona/issues/935
Forwarded: not-needed

--- a/tests/test_drvsupport.py
+++ b/tests/test_drvsupport.py
@@ -23,6 +23,7 @@ def test_geojsonseq(format):
     assert format in fiona.drvsupport.supported_drivers.keys()
 
 
+@pytest.mark.xfail(strict=False)
 @pytest.mark.parametrize(
     "driver", [driver for driver, raw in supported_drivers.items() if "w" in raw]
 )
@@ -96,6 +97,7 @@ def test_write_does_not_work_when_gdal_s
 
 # Some driver only allow a specific schema. These drivers can be
 # excluded by adding them to blacklist_append_drivers.
+@pytest.mark.xfail(strict=False)
 @pytest.mark.parametrize(
     "driver", [driver for driver, raw in supported_drivers.items() if "a" in raw]
 )
